docfx_yaml/extension.py

```diff
@@ -388,11 +388,8 @@ def extract_keyword(line):
 
 # Given lines of code, indent to left by 1 block, based on
 # amount of trailing white space of first line as 1 block.
-def indent_code_left(lines):
+def indent_code_left(lines, tab_space):
```

Comment:
Can you add a doc comment saying what these args are?

Reply:
Done.